esp/esp/program/modules/handlers/adminclass.py

```diff
@@ -396,6 +399,10 @@ def coteachers(self, request, tl, one, two, module, extra, prog):
             if cls.conflicts(teacher):
                 conflictinguser = teacher
             else:
+                if not RegistrationProfile.getLastForProgram(teacher, prog).teacher_info:
+                    lastProf = RegistrationProfile.getLastForProgram(teacher, prog)
```

Comment:
This and the previous line are the same, meaning you're making the same call twice. Why not just move this line outside of the if loop, then do `if not lastProf.teacher_info`?